### ISSUE TITLE
ArrayIndexOutOfBoundsException on reconnect

### ISSUE THREAD

**Issue by dpreussler** [dsrees/JavaPhoenixClient#10]
seeing this that takes down the whole app, not sure when it happens

```
java.lang.ArrayIndexOutOfBoundsException: length=3; index=3
at org.phoenixframework.PhxSocket$reconnectAfterMs$1.invoke(PhxSocket.kt:42)
at org.phoenixframework.PhxSocket$reconnectAfterMs$1.invoke(PhxSocket.kt:26)
at org.phoenixframework.PhxTimer.scheduleTimeout(PhxTimer.kt:31)
at org.phoenixframework.PhxChannel$1.invoke(PhxChannel.kt:71)
at org.phoenixframework.PhxChannel$1.invoke(PhxChannel.kt:6)
at org.phoenixframework.PhxTimer$scheduleTimeout$$inlined$schedule$1.run(Timer.kt:150)
at java.util.TimerThread.mainLoop(Timer.java:555)
at java.util.TimerThread.run(Timer.java:505)
```

**Comment by dsrees** [comment]
Thanks, i'll take a look.

**Comment by dpreussler** [comment]
simple issue, opened PR

**Comment by dsrees** [comment]
Resolved in 0.1.2
